doc/source/whatsnew/v0.24.0.txt

```diff
@@ -84,7 +84,7 @@ Other Enhancements
 - :meth:`Series.nlargest`, :meth:`Series.nsmallest`, :meth:`DataFrame.nlargest`, and :meth:`DataFrame.nsmallest` now accept the value ``"all"`` for the ``keep`` argument. This keeps all ties for the nth largest/smallest value (:issue:`16818`)
 - :class:`IntervalIndex` has gained the :meth:`~IntervalIndex.set_closed` method to change the existing ``closed`` value (:issue:`21670`)
 - :func:`~DataFrame.to_csv` and :func:`~DataFrame.to_json` now support ``compression='infer'`` to infer compression based on filename (:issue:`15008`)
--
+- Changed DatetimeIndex.time to also return timezone information. (:issue:`21358)
```

Comment:
Missing a closing backtick for the issue number. Can you also add an :attr: ref for DatetimeIndex.time?

Reply:
Does that look good?